docs/how-to-contribute/index.md

```diff
@@ -0,0 +1,268 @@
+# How to Contribute to this Project
+
+If you are interested in contributing to this project,
+please read our [general DSP contribution guidelines](https://docs.dasch.swiss/developers/dsp/contribution/) first.
+
+## Structure of This Project
+
+This project consists of two main parts:
+
+1. The Angular library `DSP-UI-LIB (@dasch-swiss/dsp-ui)` in `project/dsp-ui`.
+1. The demo Angular Application (Playground) in `src/app`.
+
+The Angular library contains the code that is published on NPM.
+The demo Angular application's purpose is to demonstrate the library's functionality.
+The library needs to be built first in order to use it withing the demo application.
```

Comment:
Ithink you meant "within" and not "withinG"

Reply:
resolved in f8f1fcd